src/pybind/tests/test_kaldi_pybind.py

```diff
@@ -58,6 +58,18 @@ def test_matrix_reader_writer(self):
         np.testing.assert_array_equal(value.numpy(), gold)
 
         matrix_reader.Close()
+
+        # test with context manager
+        kp_matrix[0, 0] = 20
+        with kaldi.MatrixWriter(wspecifier) as writer:
+            writer.Write('id_2', kp_matrix)
+        with kaldi.SequentialMatrixReader(rspecifier) as reader:
+            key = reader.Key()
+            self.assertEqual(key, "id_2")
```

Comment:
I would suggest using single quotes for strings for consistency.

Reply:
sure, will update now